views/discovery.hbs

```diff
@@ -18,165 +21,151 @@
+                                                    <label class="btn btn-outline-secondary" for="subject_types_supported_checkbox"> subject_types_supported </label>
 
-                                                <input type="checkbox" class="btn-check" autocomplete="off" id="token_endpoint_checkbox" name="token_endpoint" value="1" {{#if result.previously_checked.token_endpoint}} checked{{/if}}>
-                                                <label class="btn btn-outline-primary" for="token_endpoint_checkbox"> token_endpoint </label>
+                                                    <input type="checkbox" class="btn-check" autocomplete="off" id="token_endpoint_checkbox" name="token_endpoint" value="1" {{#if previous.checked.token_endpoint}} checked{{/if}}>
+                                                    <label class="btn btn-outline-secondary" for="token_endpoint_checkbox"> token_endpoint </label>
 
-                                                <input type="checkbox" class="btn-check" autocomplete="off" id="token_endpoint_auth_methods_supported_checkbox" name="token_endpoint_auth_methods_supported" value="1" {{#if result.previously_checked.token_endpoint_auth_methods_supported}} checked{{/if}}>
-                                                <label class="btn btn-outline-primary" for="token_endpoint_auth_methods_supported_checkbox"> token_endpoint_auth_methods_supported </label>
+                                                    <input type="checkbox" class="btn-check" autocomplete="off" id="token_endpoint_auth_methods_supported_checkbox" name="token_endpoint_auth_methods_supported" value="1" {{#if previous.checked.token_endpoint_auth_methods_supported}} checked{{/if}}>
+                                                    <label class="btn btn-outline-secondary" for="token_endpoint_auth_methods_supported_checkbox"> token_endpoint_auth_methods_supported </label>
 
-                                                <input type="checkbox" class="btn-check" autocomplete="off" id="userinfo_endpoint_checkbox" name="userinfo_endpoint" value="1" {{#if result.previously_checked.userinfo_endpoint}} checked{{/if}}>
-                                                <label class="btn btn-outline-primary" for="userinfo_endpoint_checkbox"> userinfo_endpoint </label>
+                                                    <input type="checkbox" class="btn-check" autocomplete="off" id="userinfo_endpoint_checkbox" name="userinfo_endpoint" value="1" {{#if previous.checked.userinfo_endpoint}} checked{{/if}}>
+                                                    <label class="btn btn-outline-secondary" for="userinfo_endpoint_checkbox"> userinfo_endpoint </label>
+                                                </div>
```

Comment:
Maybe it would be a good ideal to display the options in two columns since there are so many values? I have to scroll quite a lot when I use my laptop if I want to disable/enable the last elements in the list and click on submit.